### PR TITLE
system-variable: update variable for analyze

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  
- Other reference link(s): #47275

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -933,7 +933,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
+- 默认值：`2` v7.4.0及其之前版本默认值为 `1`
```
Comment (hfxsd) [inline system-variables.md:936]:
```suggestion
- 默认值：`2`。TiDB v7.4.0 及其之前版本默认值为 `1`。
```

File: system-variables.md [hunk 2]
```diff
@@ -1162,11 +1162,23 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 单位：线程
-- 默认值：`4`
+- 默认值：`2`  v7.4 版本及其之前版本默认值为 `4`
```
Comment (hfxsd) [inline system-variables.md:1165]:
```suggestion
- 默认值：`2`。TiDB v7.4.0 及其之前版本默认值为 `4`。
```

File: system-variables.md [hunk 3]
```diff
@@ -1162,11 +1162,23 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+### `tidb_build_sampling_stats_concurrency` <span class="version-mark">从 v7.5 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 单位：线程
+- 默认值：`2`
+- 取值范围：`[1, 256]`
+- 这个变量用来设置 ANALYZE 过程中采样并发度。
```
Comment (hfxsd) [inline system-variables.md:1179]:
```suggestion
- 这个变量用来设置 `ANALYZE` 过程中的采样并发度。
```

**Review by qw4990 — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

File: system-variables.md [hunk 4]
```diff
@@ -1162,11 +1162,23 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 单位：线程
-- 默认值：`4`
+- 默认值：`2`。TiDB v7.4.0 及其之前版本默认值为 `4`。
 - 取值范围：`[1, 256]`
 - 这个变量用来设置 ANALYZE 语句执行时并发度。
 - 当这个变量被设置得更大时，会对其它的查询语句执行性能产生一定影响。
 
+### `tidb_build_sampling_stats_concurrency` <span class="version-mark">从 v7.5 版本开始引入</span>
```
Comment (qiancai) [inline system-variables.md:1170]:
```suggestion
### `tidb_build_sampling_stats_concurrency` <span class="version-mark">从 v7.5.0 版本开始引入</span>
```

File: system-variables.md [hunk 5]
```diff
@@ -1162,11 +1162,23 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 单位：线程
+- 默认值：`2`
+- 取值范围：`[1, 256]`
+- 这个变量用来设置 `ANALYZE` 过程中的采样并发度。
+- 当这个变量被设置得更大时，会对其它的查询语句执行性能产生一定影响。
```
[the remark this answers is not in the document]
Comment (hawkingrei) [reply system-variables.md:1180]: 设置得越大，对事例影响就越大

**Review by qiancai — APPROVED** [review]

**Pull request merged**